bpf/generictracer/protocol_http.h

```diff
@@ -257,8 +263,28 @@ static __always_inline void finish_http(http_info_t *info, pid_connection_info_t
 
         // bpf_dbg_printk("Terminating trace for pid=%d", pid_from_pid_tgid(pid_tid));
         // dbg_print_http_connection_info(&info->conn_info); // commented out since GitHub CI doesn't like this call
-        bpf_map_delete_elem(&ongoing_http, pid_conn);
+        // Don't delete requests that weren't delayed, we might be receiving still more packets, for
+        // example SSL.
+        if (info->delayed) {
+            bpf_map_delete_elem(&ongoing_http, pid_conn);
+        }
+    }
+}
+
+static __always_inline void force_finish_http(http_info_t *info, pid_connection_info_t *pid_conn) {
+    if (info->submitted) {
+        return;
+    }
+
+    if (!high_request_volume) {
+        if (!http_info_complete(info)) {
+            info->resp_len = 0;
+            info->end_monotime_ns = bpf_ktime_get_ns();
+            info->status = 409;
```

Comment:
I'm setting the status as 499 when we haven't yet completed the request but we are pushing another. This is typically caused by a client timeout and retry and NGINX sets 499 (although it's not officially set)